File: iOS/UserInfo.swift
//
//  userinfo.swift
//  ustb_choose_course_system
//
//  Created by kalen blue on 15-8-9.
//  Copyright (c) 2015年 kalen blue. All rights reserved.
//

import Foundation

let userInfo = kalen.app.UserInfo()

extension kalen.app{
    class UserInfo{
        var username:String = "";
        var password:String = "";
        
        class func getInstance() -> UserInfo{
            
            return userInfo
        }
        
        init(){
            
        }
        
    }
}

File: PC/userinfo.cpp
#include "userinfo.h"
UserInfo* UserInfo::userInfo = NULL;

UserInfo::UserInfo()
{

}

/**
 * @brief UserInfo::getInstance
 * @return
 */
UserInfo* UserInfo::getInstance (){
    if(userInfo == NULL){
        userInfo = new UserInfo;
    }
    return userInfo;
}

